bug: if feature not register, export will panic

To Reproduce:

```
$ oomcli init
$ oomcli export --feature account.state,account.account_age_days --unix-milli 1950236233000
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16dd732]

goroutine 1 [running]:
github.com/oom-ai/oomstore/oomcli/cmd.printExportResultInASCIITable(0x0)
        /home/lianxm/github.com/oomstore/oomcli/cmd/export_helper.go:64 +0x52
github.com/oom-ai/oomstore/oomcli/cmd.printExportResult(0xc0001359c0, {0x1aafd72, 0xc000135a00})
        /home/lianxm/github.com/oomstore/oomcli/cmd/export_helper.go:36 +0x6a
github.com/oom-ai/oomstore/oomcli/cmd.export({0x1da49c8, 0xc00004e068}, 0xc0003398a0, {{0xc00013ece0, 0x2, 0x2}, 0x1c613234d28, 0x0}, {0x1aafd72, 0xb})
        /home/lianxm/github.com/oomstore/oomcli/cmd/export_helper.go:25 +0xd1
github.com/oom-ai/oomstore/oomcli/cmd.glob..func10(0x2b23b60, {0x1aa5b67, 0x4, 0x4})
        /home/lianxm/github.com/oomstore/oomcli/cmd/export.go:26 +0x108
github.com/spf13/cobra.(*Command).execute(0x2b23b60, {0xc000135080, 0x4, 0x4})
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x2b21ae0)
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
github.com/oom-ai/oomstore/oomcli/cmd.Execute()
        /home/lianxm/github.com/oomstore/oomcli/cmd/root.go:32 +0x25
main.main()
        /home/lianxm/github.com/oomstore/oomcli/main.go:6 +0x17
```

--- a/oomagent/test/test_channel_export.sh
+++ b/oomagent/test/test_channel_export.sh
@@ -10,9 +10,44 @@ oomcli push --entity-key 1 --group user_fake_stream --feature f1=10
 oomcli snapshot user_fake_stream
 
 unix_milli=${3:-$(perl -MTime::HiRes=time -E 'say int(time * 1000)')}
-echo $unix_milli
 
-case1() {
+oomagent_export_no_register_feature() {
+    case="export all no register feature"
+    arg=$(cat <<-EOF
+    {
+       "features": ["a.b","a.c"],
+       "unix_milli": $unix_milli
+    }
+EOF
+)
+    actual=$(testgrpc ChannelExport <<<"$arg" 2>&1 || true)
+    expected='
+ERROR:
+  Code: Internal
+  Message: invalid feature names [a.b a.c]
+'
+    assert_eq "$case" "$expected" "$actual"
+}
+
+oomagent_export_has_no_register_feature() {
+    case="export has no register feature"
+    arg=$(cat <<-EOF
+    {
+       "features": ["account.state","account.a"],
+       "unix_milli": $unix_milli
+    }
+EOF
+)
+    actual=$(testgrpc ChannelExport <<<"$arg" 2>&1 || true)
+    expected='
+ERROR:
+  Code: Internal
+  Message: invalid feature names [account.a]
+'
+    assert_eq "$case" "$expected" "$actual"
+}
+
+oomagent_export_no_features() {
     case="export and no features"
     arg=$(cat <<-EOF
     {
@@ -26,7 +61,7 @@ EOF
     assert_json_eq "$case" "$expected" "$actual"
 }
 
-case2() {
+oomagent_export_one_features() {
     prefix="export some features"
     arg=$(cat <<-EOF
     {
@@ -59,7 +94,7 @@ EOF
     assert_json_eq "$case" "$(sort <<<"$expected_rows")" "$actual_rows"
 }
 
-case3() {
+oomagent_export_multi_features() {
     prefix="export some features"
     arg=$(cat <<-EOF
     {
@@ -92,6 +127,12 @@ EOF
     assert_json_eq "$case" "$(sort <<<"$expected_rows")" "$actual_rows"
 }
 
-case1
-case2
-case3
+main() {
+    oomagent_export_no_register_feature
+    oomagent_export_has_no_register_feature
+    oomagent_export_no_features
+    oomagent_export_one_features
+    oomagent_export_multi_features
+}
+
+main

--- a/oomcli/test/test_export.sh
+++ b/oomcli/test/test_export.sh
@@ -19,6 +19,20 @@ t3=${1:-$(perl -MTime::HiRes=time -E 'say int(time * 1000)')}
 oomcli push --entity-key 2 --group user-click --feature last_5_click_posts=1,2,3,4,5 --feature number_of_user_starred_posts=10
 t4=${1:-$(perl -MTime::HiRes=time -E 'say int(time * 1000)')}
 
+oomcli_export_no_register_feature() {
+    case="export all no register feature"
+    actual=$(oomcli export --feature a.b,a.c --unix-milli $t0 2>&1 || true)
+    expected='Error: failed exporting features: invalid feature names [a.b a.c]'
+    assert_eq "$case" "$expected" "$actual"
+}
+
+oomcli_export_has_no_register_feature() {
+    case="export has no register feature"
+    actual=$(oomcli export --feature user-click.last_5_click_posts,user-click.a --unix-milli $t0 2>&1 ||true)
+    expected='Error: failed exporting features: invalid feature names [user-click.a]'
+    assert_eq "$case" "$expected" "$actual"
+}
+
 oomcli_export_push_feature() {
     case="push feature"
     expected='user,user-click.last_5_click_posts,user-click.number_of_user_starred_posts
@@ -77,6 +91,8 @@ oomcli_export_batch_and_stream() {
 }
 
 main() {
+    oomcli_export_no_register_feature
+    oomcli_export_has_no_register_feature
     oomcli_export_push_feature
     oomcli_export_update_feature
     oomcli_export_batch

--- a/pkg/oomstore/export.go
+++ b/pkg/oomstore/export.go
@@ -38,9 +38,6 @@ func (s *OomStore) ChannelExport(ctx context.Context, opt types.ChannelExportOpt
 	if err != nil {
 		return nil, err
 	}
-	if len(features) == 0 {
-		return nil, nil
-	}
 
 	if len(features) != len(opt.FeatureNames) {
 		invalid := features.FindMissingFeatures(opt.FeatureNames)

